diff --git a/node_modules/@stryker-mutator/instrumenter/dist/src/parsers/svelte-parser.js b/node_modules/@stryker-mutator/instrumenter/dist/src/parsers/svelte-parser.js
index 587648a..4914c34 100644
--- a/node_modules/@stryker-mutator/instrumenter/dist/src/parsers/svelte-parser.js
+++ b/node_modules/@stryker-mutator/instrumenter/dist/src/parsers/svelte-parser.js
@@ -2,9 +2,10 @@ import { notEmpty } from '@stryker-mutator/util';
 import { satisfies } from 'semver';
 import { AstFormat } from '../syntax/index.js';
 import { PositionConverter } from '../util/index.js';
+import { walk } from 'estree-walker';
 const MIN_SVELTE_VERSION = '>=3.30';
 export async function parse(text, fileName, context) {
-    const { parse: svelteParse, walk, preprocess, VERSION } = await import('svelte/compiler');
+    const { parse: svelteParse, preprocess, VERSION } = await import('svelte/compiler');
     if (!satisfies(VERSION, MIN_SVELTE_VERSION)) {
         throw new Error(`Svelte version ${VERSION} not supported. Expected: ${MIN_SVELTE_VERSION} (processing file ${fileName})`);
     }
